contracts/governance/AxelarServiceGovernance.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { IAxelarServiceGovernance } from '../interfaces/IAxelarServiceGovernance.sol';
+import { InterchainGovernance } from './InterchainGovernance.sol';
+import { MultisigBase } from '../auth/MultisigBase.sol';
+
+contract AxelarServiceGovernance is InterchainGovernance, MultisigBase, IAxelarServiceGovernance {
+    enum ServiceGovernanceCommand {
+        ScheduleTimeLockProposal,
+        CancelTimeLockProposal,
+        ApproveMultisigProposal,
+        CancelMultisigApproval
```

Comment:
```suggestion
        ScheduleCosignerProposal,
        CancelCosignerApproval
```
Perhaps?

Reply:
It seems that you either can have a `InterchainGovernance` that can only be operated by the Axelar governance, or AxelarServiceGovernance that can be executed by the Axelar governance but can also require the approval of some admins. Is there an option that I am missing for just admin executions (which if i understand correctly should be used for the ITS flow limits). 
Also, what is the point of `InterchainGovernance` since `AxelarServiceGovernance` is just more stuff. Especially since there is no way to know as the target contract if the call was approved by admins or timelocked.

Reply:
@Foivos #198 has the admin only approval for mint limits.

Target contract shouldn't care about the specific policy with which the execution was initiated. Target contract should have distinct roles if it needs different policies.